src/main.js

```diff
@@ -92,12 +96,31 @@ export class Draggable {
             const { which } = e;
 
             if ((which && which > 1) || this._ignoreMouse) {
+                bind(this.document, "contextmenu", preventDefault);
+                
+                this.cancelDrag(e);
+
+                this._cancelHandler(e);
+
                 return;
             }
 
-            bind(this.document, "mousemove", this._mousemove);
-            bind(this.document, "mouseup", this._mouseup);
-            this._pressHandler(e);
+            if (!this._clickMoveClick) {
+                bind(this.document, "mouseup", this._mouseup);
+                bind(this.document, "mousemove", this._mousemove);
+                bind(this.document, "contextmenu", preventDefault);
+                this._pressHandler(e);
+            } else {
+                if (this._isDragging) {
+                    bind(this.document, "mouseup", this._mouseup);
+                } else {
+                    bind(this.document, "mousemove", this._mousemove);
```

Comment:
We can add a comment about why `isDragging` can be `true` here - in the general case, it is not possible for the "start" event to be triggered after a "move" action has taken place, but we want exactly this here, so an explanation would be helpful.